src/main/java/fi/otavanopisto/kuntaapi/server/security/SecurityFilter.java

```diff
@@ -48,6 +48,9 @@ public void filter(ContainerRequestContext requestContext) {
       return;
     }
     
+    if (true)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2589)